libfabric.map.cpp.in

```diff
@@ -13,5 +13,9 @@ FABRIC_1.0 {
 		fi_param_get;
 		fi_getparams;
 		fi_freeparams;
+#ifdef FI_DIRECT_ENABLED
+#include "prov/@PROVIDER_DIRECT@/provider.map"
+#endif
 	local: *;
 };
```

Comment:
@shefty I think an easy way to deal with 1740 is to have ```FABRIC_1.0``` in the file name for the provider's part of the version script.  That way if we add newer version sections in the future, we can just keep extending by having the provider add a ```FABRIC_2.0```, etc.